src/auth_client_demo_assets/svelte/src/components/LoggedOut.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
+  import { auth } from "../stores/auth";
+</script>
+
+<div class="container">
+  <h1>Internet Identity Client</h1>
+  <h2>You are not authenticated</h2>
+  <p>To log in, click this button!</p>
+  <button type="button" id="loginButton" on:click={$auth.login}>Log in</button>
```

Comment:
Just FYI: I was confused at first, did not checked that the component was called `LoggedOut` and actually contains a `Log in` action. Not call to action, just in case.